docker/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM python:3.10.6-slim-bullseye
+ARG VERSION
+RUN pip install --no-cache mlflow==$VERSION
```

Comment:
It complains about not having `psycopg2` installed when using with **postgres**

Reply:
@Rusteam can you create an issue with a reproducible example, since this PR has already been merged? With the reproducible example I can identify and rectify any missing OS level dependencies.

@harupy do you think it is better to add this to the requirements.txt for mlflow?